src/mount/sauna_client.cc

```diff
@@ -836,6 +854,7 @@ AttrReply getattr(Context &ctx, Inode ino) {
 	Attributes attr;
 	char attrstr[256];
 	int status;
+	bool fromcache;
```

Comment:
Consider using camelCase for variable name.
eg: 
```
bool fromCache;
```

Reply:
Done.